std/container/dlist.d

```diff
@@ -62,6 +62,13 @@ module std.container.dlist;
     rs.popFront();
     nl.remove(rs);
     assert(equal(nl[], [1]));
+
+    () @nogc {
+        auto rn = nl[];
+        auto el = nl.front;
```

Comment:
Should probably also test `.back`

Reply:
I'm not sure that this test is even required. I mean, if the code compiles with `@nogc:` that means that the functions that were annotated have passed compilation. I would simply drop this test.